diagnostic.md

```diff
@@ -157,7 +188,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+All divs besides divOne will float to the left, flush left to eachother inline
+touching the left side of the viewport. divOne will float right touching the
+right side of the viewport.
 <!-- your answer ends here -->
```

Comment:
Close. .divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).